artemis/config.py

```diff
@@ -404,6 +404,11 @@ class WordPressBruter:
                 "www123, when testing www.projectname.example.com.",
             ] = get_config("WORDPRESS_BRUTER_STRIPPED_PREFIXES", default="www", cast=decouple.Csv(str))
 
+        class DomainExpirationScanner:
+            DOMAIN_EXPIRATION_ALERT_IN_DAYS: Annotated[
```

Comment:
I would prefer `DOMAIN_EXPIRATION_TIMEFRAME_DAYS` (for other variables we write the unit as a direct suffix, not 'in days')